person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    age >= 18
+  end
+
+  def can_use_services?
+    of_age? || parent_permission
+  end
```

Comment:
- [ ] The `of_age?` method is expected to be a `private` method. Kindly do that.

  SUGGESTION: You can rearrange your methods like so:
  ```
  def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age?
    age >= 18
  end
  ```